### PR TITLE
Revert usage of cyndi data

### PR DESCRIPTION
**Pull request by semtexzv**
Revert usage of project cyndi data. Keep changes regarding primary keys.

### CODE REVIEW

File: manager/controllers/utils.go [hunk 1]
```diff
@@ -205,24 +202,19 @@ func ApplySearch(c *gin.Context, tx *gorm.DB, searchColumns ...string) *gorm.DB
...
-		Select("h.id::text")
-
-	for _, t := range tags {
-		matches := tagRegex.FindStringSubmatch(t)
-		tagJSON := fmt.Sprintf(`[{"namespace":"%s", "key": "%s", "value": "%s"}]`, matches[1], matches[2], matches[3])
-		subq = subq.Where(" h.tags @> ?::jsonb", tagJSON)
-	}
+		Table("system_tags st").
+		Select("DISTINCT st.system_id").
+		Where("st.tag in (?)", tags).
```
Comment (Josca) [inline manager/controllers/utils.go:214]: It seems it does not work correctly for multiple tags. It should return only systems with all tags included in query tags array. Please add these tests to the `TestSystemsTags`:
~~~go
func TestSystemsTagsUnknown(t *testing.T) {
	utils.SkipWithoutDB(t)
	core.SetupTestEnvironment()

	w := httptest.NewRecorder()
	req, _ := http.NewRequest("GET", "/?tags=ns1/k1=unknown", nil)
	core.InitRouterWithPath(SystemsListHandler, "/").ServeHTTP(w, req)

	assert.Equal(t, 200, w.Code)
	var output SystemsResponse
	ParseReponseBody(t, w.Body.Bytes(), &output)
	assert.Equal(t, 0, len(output.Data))
}

func TestSystemsTagsMultiple1(t *testing.T) {
	utils.SkipWithoutDB(t)
	core.SetupTestEnvironment()

	w := httptest.NewRecorder()
	req, _ := http.NewRequest("GET", "/?tags=ns1/k1=val1&tags=ns1/k2=val2", nil)
	core.InitRouterWithPath(SystemsListHandler, "/").ServeHTTP(w, req)

	assert.Equal(t, 200, w.Code)
	var output SystemsResponse
	ParseReponseBody(t, w.Body.Bytes(), &output)
	assert.Equal(t, 1, len(output.Data))
	assert.Equal(t, "00000000-0000-0000-0000-000000000001", output.Data[0].ID)
}

func TestSystemsTagsMultiple2(t *testing.T) {
	utils.SkipWithoutDB(t)
	core.SetupTestEnvironment()

	w := httptest.NewRecorder()
	req, _ := http.NewRequest("GET", "/?tags=ns1/k1=val1&tags=ns1/k2=unknown", nil)
	core.InitRouterWithPath(SystemsListHandler, "/").ServeHTTP(w, req)

	assert.Equal(t, 200, w.Code)
	var output SystemsResponse
	ParseReponseBody(t, w.Body.Bytes(), &output)
	assert.Equal(t, 0, len(output.Data))
}
~~~

File: database_admin/migrations/031_cyndi_tags_revert.down.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+DROP TABLE IF EXISTS system_tag CASCADE;
+DROP TABLE IF EXISTS tag CASCADE;
```
Comment (Josca) [inline database_admin/migrations/031_cyndi_tags_revert.down.sql:2]: Line end.

**Review by Josca — CHANGES_REQUESTED** [review]

File: manager/controllers/utils.go (hunk 1 above)
Comment (semtexzv) [reply manager/controllers/utils.go:214]: Oh, yes, you're right, there was a bugfix after implementing this

**Comment by semtexzv** [comment]
Just disabling the current implementation for environments where not supported is enough. Closing.

**Pull request closed without merging**